docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,518 @@
+# corresponding data values, G_t
+Γy = convert(Array, Diagonal([100.0, 5.0, 30.0]))
+μ = zeros(length(G_t))
+
+# Add noise
+for i in 1:n_samples
+    y_t[:, i] = G_t .+ rand(MvNormal(μ, Γy))
+end
+
+truth = Observations.Observation(y_t, Γy, data_names)
+truth_sample = truth.mean 
+```
+
+#### Perform ensemble Kalman inversion
+
+We sample the initial ensemble from the prior and create the
+`EnsembleKalmanProcess` object as an ensemble Kalman inversion (EKI) algorithm
+using the `Inversion()` keyword. We also use the `DataMisfitController()`
+learning rate scheduler. The number of ensemble members must be larger than the dimension of the parameter space to ensure a full rank ensemble covariance.
+
+```julia
+N_ens = 50 # number of ensemble members
+N_iter = 8 # number of EKI iterations
+# initial parameters: n_params x N_ens
+initial_params = construct_initial_ensemble(rng, priors, N_ens)
+ekiobj = EnsembleKalmanProcess(
+    initial_params,
+    truth_sample,
+    truth.obs_noise_cov,
+    Inversion(),
+    scheduler=DataMisfitController()
+)
+```
+
+We perform the inversion loop. Remember that within calls to `get_ϕ_final` the
+EKP transformations are applied, thus the ensemble that is returned will be the
+gamma distribution parameters that can be used to run the forward model, rather
+than their corresponding values in the unconstrained space where the EKI takes
+place. Each ensemble member is stored as a column and therefore for uses such as plotting one needs to reshape to the desired dimension.
+
+```julia
+# Initialize a ParticleDistribution with dummy parameters. The parameters 
+# will then be set within `run_dyn_model`
+dummy = ones(n_params)
+dist_type = ParticleDistributions.GammaPrimitiveParticleDistribution(dummy...)
+model_settings = DynamicalModel.ModelSettings(kernel, dist_type, moments, tspan)
+# EKI iterations
+for n in 1:N_iter
+    # Return transformed parameters in physical/constrained space
+    ϕ_n = get_ϕ_final(priors, ekiobj)
+    # Evaluate forward map
+    G_n = [DynamicalModel.run_dyn_model(ϕ_n[:, i], model_settings) for i in 1:N_ens]
+    G_ens = hcat(G_n...)  # reformat
+    EnsembleKalmanProcesses.update_ensemble!(ekiobj, G_ens)
+end
+```
+
+#### Visualize and store the results of the calibration
+
+The optimal parameter vector determined by the ensemble Kalman inversion is the
+ensemble mean of the particles after the last iteration, which is printed to
+standard output. An output directory is created, where a file
+`cloudy_calibrate_results.jld2` is stored, which contains all parameters and
+model output from the ensemble Kalman iterations (both as
+`DataContainers.DataContainer` objects), the mean and one sample of the
+synthetic observations, as well as the true prameters and their priors. In
+addition, an animation is produced that shows the evolution of the ensemble of
+particles over subsequent iterations of the optimization, both in the
+computational (unconstrained) and physical (constrained) spaces.
```

Comment:
You can put the visualization here if you like? Would be the right place for it. you just include the `.gif` as you would do a figure